src/gdata.h

```diff
@@ -29,40 +29,34 @@
 
 ******************************************************************************/
 
-#define KPP_VERSION "2.2.3"
+#define KPP_VERSION "3.0.0"
```

Comment:
I updated the version number here since the new version will be a major release (also containing the autoreduce code from @jimmielin).  If you would like me to change this we can (maybe to 3.0.0-rc.0 or something similar).

Reply:
We should certainly aim for version number 3.0.0 for our major release!
However, I think it would be good to go step by step, and assign version
numbers also to intermediate versions. According to semver
(https://semver.org), the second number increases when new functionality
is added. More specifically, I would like to assign version number 2.3.0
to the code after the merge with the MECCA branch. Next, we could create
2.4.0 after merging the geos-chem-updates. What do you think?

Reply:
This sounds OK.  The only thing is that in the https://geoschem/KPP repository, I was using version 2.3.3_gc as the latest version.  Wondering if we want to bump up to 2.4.0 after merge w/ MECCA and then 2.5 for GEOS-Chem to avoid confusion.  But I am flexible.

Reply:
Yes. Let's assign version number 2.4.0 in the dev branch after
fixing the ICNTRL(15) bug but before merging it with GEOS-Chem.

Reply:
I think the dev branch is now ready to be called 2.4.0-rc.0. If you agree, I
will create this version now:

- adjust KPP_VERSION in src/gdata.h
- mention the new version in CHANGELOG.md
- create corresponding git tags

I would like to perform a couple of tests with this version inside my
MECCA module. After (maybe) some bug fixes, we can create version 2.4.0
in the main branch.

Reply:
I updated the version number to 2.4.0 and fixed a couple of things.  See PR #29.  Feel free to push bug fixes there.

Reply:
Thanks! I hope to finish my tests during the next week. If you want to
go ahead with PR #28 while I'm still testing, I can perform my tests in
a separate relese branch...